dist/src/collections.js

```diff
@@ -25,8 +25,10 @@ var addCollectionsApi = function addCollectionsApi(matroid) {
   matroid.createCollection = function (name, url, sourceType) {
     var _this2 = this;
 
+    var configs = arguments.length > 3 && arguments[3] !== undefined ? arguments[3] : {};
```

Comment:
Maybe let's go with a `typeof` check instead of not `undefined` to make sure that it's an object.